locales/templates/app.pot

```diff
@@ -305,6 +305,140 @@ msgctxt "preference_about"
 msgid "About %1$s"
 msgstr ""
 
+#. Header for the list of installed search engines
+msgctxt "preference_search_installed_search_engines"
+msgid "Installed search engines"
+msgstr ""
+
+#. Action for restoring the default list of search engines
+msgctxt "preference_search_restore"
+msgid "Restore default search engines"
+msgstr ""
+
+#. Action for adding an additional search engine to the list
+msgctxt "preference_search_add"
+msgid "Add another search engine"
+msgstr ""
+
+msgctxt "preference_search_remove"
+msgid "Remove"
+msgstr ""
+
+#. Clickable text in tutorial to open a dialog to manually add a search engine
+msgctxt "tutorial_search_manually_add"
+msgid "MANUALLY ADD"
+msgstr ""
+
+msgctxt "tutorial_search_cancel"
+msgid "CANCEL"
+msgstr ""
+
+msgctxt "tutorial_search_title"
+msgid "Add search engine"
+msgstr ""
+
+msgctxt "tutorial_search_step1"
+msgid "1. Go to the search engine homepage."
+msgstr ""
+
+msgctxt "tutorial_search_step2"
+msgid "2. Tap into the search engine's search field."
```

Comment:
Please use proper apostrophes (’ instead of ') in new strings. I've seen Firefox for iOS going through a [complete review](https://github.com/mozilla-l10n/firefoxios-l10n/pull/39), and that invalidated all existing translations. I'd like to avoid that.

Reply:
I'll update these and re-push a PR.